styles/style.css

```diff
@@ -0,0 +1,80 @@
+h1 {
+  font-family: 'Noto Serif JP', sans-serif;
+}
+
+ul {
```

Comment:
great job! i feel like it's very DRY to do this / to put this at the top so it applies to all ul vs editing one by one